src/StackHCI/test/Add-AzStackHCIVMAttestation.Tests.ps1

```diff
@@ -0,0 +1,29 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Add-AzStackHCIVMAttestation'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'Add-AzStackHCIVMAttestation.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'Add-AzStackHCIVMAttestation' {
+    It 'VMName' -skip {
```

Comment:
Please add test case for each cmdlet.

Reply:
Hi @BethanyZhou, our cmdlets consist of some generated and custom cmdlets. These custom cmdlets are already existing in powershell today. It is not possible to add tests for them because they need to be run from our HCI OS device. I have added the tests for the generated cmdlets.